tests/batch/autoscaler_test.py

```diff
@@ -75,3 +77,16 @@ def test_run(mock_sensu, mock_running, mock_time, mock_sleep, dry_run):
     assert batch_obj.autoscaler.run.call_args_list == [mock.call(dry_run=dry_run) for i in range(3)]
     assert mock_sleep.call_args_list == [mock.call(499), mock.call(287), mock.call(400)]
     assert mock_sensu.call_count == 6
+
+
+@mock.patch('clusterman.batch.autoscaler.AutoscalerBatch.running', new_callable=mock.PropertyMock)
+@pytest.mark.parametrize('exc', [PoolConnectionError, EndpointConnectionError(endpoint_url='')])
+def test_run_connection_error(mock_running, exc):
+    batch_obj = batch()
+    batch_obj._autoscale = mock.Mock(side_effect=exc)
+    mock_running.side_effect = [True, True, False]
+
+    batch_obj.run()
+
+    # exceptions raised did prevent subsequent calls to autoscale
+    assert batch_obj._autoscale.call_count == 1
```

Comment:
I guess this test doesn't run in Travis (because it's not a yelpy env) but this should be 2, right?